src/navigation/ProductDetailsNavigator.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {createNativeStackNavigator} from '@react-navigation/native-stack'
+import {ProductDetailsScreen} from '../screens'
+
+const Stack = createNativeStackNavigator()
+
+export const ProductDetailsNavigator = () => {
```

Comment:
I think it is redundant to have additional navigator for product details since it has only one screen

Reply:
Got it.
I was able to implement only in such a way. Will try again.